config/application.rb

```diff
@@ -46,8 +46,8 @@ class Application < Rails::Application
     config.services = YAML.load_file(Rails.root.join('config/services.yml'))
 
     # Versions of UMM for the different metadata types MMT is on
-    config.umm_c_version = 'vnd.nasa.cmr.umm+json; version=1.15.3'
-    config.umm_var_version = 'vnd.nasa.cmr.umm+json; version=1.7'
```

Comment:
This needs to be 1.7

Reply:
must've happened when I pulled in master